modules/sdk/src/main/scala/org/tessellation/sdk/http/routes/SnapshotRoutes.scala

```diff
@@ -91,6 +91,23 @@ final case class SnapshotRoutes[F[_]: Async: KryoSerializer, S <: Snapshot[_, _]
           }
         }.getOrElse(NotFound())
 
+    case GET -> Root / "hash" / SnapshotOrdinalVar(ordinal) :? FullSnapshotQueryParam(fullSnapshot) =>
+      import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
+
+      if (!fullSnapshot) {
+        snapshotStorage.getHash(ordinal).flatMap {
```

Comment:
you can use `.flatMap(Ok(_))` as you did few lines below

Reply:
Changed

```scala
flatMap {
  Ok(_)
}
```

to `.flatMap(Ok(_))`.